Cabal/src/Distribution/Simple/Program/GHC.hs

```diff
@@ -227,8 +227,14 @@ normaliseGhcArgs (Just ghcVersion) PackageDescription{..} ghcArgs
                   , "keep-going" -- try harder, the build will still fail if it's erroneous
                   , "print-axiom-incomps" -- print more debug info for closed type families
                   ]
+              , from
+                  [9, 10]
+                  [ "diagnostics-as-json"
+                  , "print-error-index-links"
+                  , "break-points"
+                  ]
               ]
-          , flagIn . invertibleFlagSet "-d" $ ["ppr-case-as-let", "ppr-ticks"]
+          , flagIn . invertibleFlagSet "-d" ["ppr-case-as-let", "ppr-ticks"]
```

Comment:
Did you test this?

Reply:
no, i didn't, it is obviously erroneous, you're right. will adjust.